src/segments/project.go

```diff
@@ -95,9 +96,9 @@ func (n *Project) Init(props properties.Properties, env platform.Environment) {
 			Fetcher: n.getCargoPackage,
 		},
 		{
-			Name:    "poetry",
+			Name:    "python",
```

Comment:
this is a breaking change, can we duplicate this entry so it works for `python` and `poetry` still?

Reply:
Unfortunately, I don't think this is possible at the moment. As I understand, the project segment detects a specific file (or file type) and passes it to the appropriate "fetcher". That is, if I'm not mistaken, more than one fetcher cannot be responsible for the same file (in our case it is `pyproject.toml`).
But I doubt that this is a breaking change. The change does not seem to break anything, but adds the ability to read data according to a different standard. Yes, the name of the fetcher changes from "Poetry" to "Python", but as far as I know nothing is "bound" to it (other than what I already changed).

Reply:
The `Name` is exposed as `.Type` in the template. So if people have templates matching `poetry` as value, it will break. So we need this in the release notes. It's a breaking change in that case. I'll adjust that.